packages/tests-postman/index.js

```diff
@@ -24,7 +27,6 @@ program
     './collections/reference-credentials.json'
   );
 
-// disable-eslint-next-line program
 //     .option('-s, --service <file>', 'use the specified interop test collection', './collections/interop-credentials.json');
```

Comment:
do we need to keep those lines in and commented?
If they're not being used we should just remove them.

Reply:
none from my side

Reply:
These dead options have been removed.